src/styles/calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  gap: 1px;
+  padding: 10px;
+  box-sizing: border-box;
+  list-style: none;
+  flex-wrap: wrap;
+  width: 707px;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    width: $cell-side;
+    height: $cell-side;
+    border: solid 1px #000;
+    box-sizing: border-box;
+    transition: transform, background-color, 0.5s;
+
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20px);
+    }
+  }
+
+  $daysCounter: 0;
+  @each $day in $weekDays {
+    &__start-day--#{$day} :first-child {
+      margin-left: $cell-side-and-gap * $daysCounter;
+      $daysCounter: $daysCounter + 1;
```

Comment:
wow, i even didnt know that its can work like this
but more easier way 
![image](https://github.com/mate-academy/layout_calendar/assets/96265844/b2cd4700-b96c-4d6b-af0e-eb6172331f12)
![image](https://github.com/mate-academy/layout_calendar/assets/96265844/160152f8-66ec-4b0a-91d4-79c8a878e742)